manager/integration/tests/test_ha.py

```diff
@@ -233,6 +233,42 @@ def ha_salvage_test(client, core_api, # NOQA
 
     cleanup_volume(client, volume)
 
+    # test auto salvage when revision counter enabled ordisabled.
+    # One for revision counter enabled case.
+    test_ha_salvage_autosalvage()
```

Comment:
`ha_salvage_test` is a common function. You need to call this common function in the test functions rather than in a reverse way.